docs/getting-started/install-configure.md

```diff
@@ -5,44 +5,28 @@ weight: 2
 indent: true
 ---
 
-# Choosing Hosted or Self-Hosted Crossplane
+# Choosing Your Crossplane Distribution
 
 Users looking to use Crossplane for the first time have two options available to
-them today. The first way is to use a hosted Crossplane service like [Upbound
-Cloud][Upbound Cloud]. Alternatively, users looking for some more
-flexibility can install Crossplane into their own Kubernetes cluster.
+them today. The first way is to use the upstream distribution of Crossplane 
+which is maintained and released by the community. 
```

Comment:
Nit: Do we want to call upstream a distribution? I think of distributions of adding extra 'stuff' (like UXP adds the ability to work with Upbound) or otherwise being a divergence from 'upstream' personally, but I guess it's not exact.

Reply:
"Upstream Crossplane" feels weird to me in this context to - a bit like using "upstream" to refer to the place in the stream where you're currently standing. Perhaps "vanilla Crossplane", or just unqualified "Crossplane"?

Reply:
Addressed with this. PTAL https://github.com/crossplane/crossplane/pull/2819/commits/8039a4a6bbab4725624bdc17cb2f5303bbf8b623